src/main/java/com/minecolonies/colony/ColonyView.java

```diff
@@ -45,6 +43,8 @@
 
     //  Citizenry
     private Map<Integer, CitizenDataView> citizens = new HashMap<>();
+    final private Map<Integer, WorkOrderView> workOrders = new HashMap<>();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AModifiersOrderCheck)